src/core_plugins/kibana/public/dashboard/index.js

```diff
@@ -146,8 +157,12 @@ uiRoutes
 FeatureCatalogueRegistryProvider.register(() => {
```

Comment:
I believe we can inject `i18n` service here.

Reply:
it looks like we can't